### PR TITLE
feat: Idpay migration to v3

### PR DESCRIPTION
**Pull request by mamari90**
### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [x] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

#### Has This Been Tested?

- [x] Yes
- [ ] No

### Other information
Target: 
```

```

#### How Has This Been Tested?

- [ ] Unit Test Suite
- [ ] Integration Test Suite
- [ ] Api Tests
- [ ] Load Tests

[Link to test results](https://pagopa.atlassian.net/browse/RTD-XXXX)

---

### CODE REVIEW

**Review by diegolagospagopa — APPROVED** [review]
LGTM

File: src/domains/idpay-common/04_cdn.tf [hunk 1]
```diff
@@ -255,7 +256,6 @@ module "selfcare_welfare_cdn" {
 
   hostname              = format("selfcare.%s", data.terraform_remote_state.core.outputs.dns_zone_welfare_name)
```
Comment (fmlisco) [inline src/domains/idpay-common/04_cdn.tf:257]: interpolation

File: src/domains/idpay-common/04_cdn.tf [hunk 2]
```diff
@@ -246,7 +247,7 @@ resource "azurerm_resource_group" "rg_welfare" {
 
 
 module "selfcare_welfare_cdn" {
-  source = "git::https://github.com/pagopa/azurerm.git//cdn?ref=v4.2.1"
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cdn?ref=v6.18.0"
 
   name                = format("welfare-selfcare-%s", var.env_short)
```
Comment (fmlisco) [inline src/domains/idpay-common/04_cdn.tf:252]: interpolation

File: src/domains/idpay-app/terraform.sh [hunk 3]
```diff
@@ -1,46 +1,192 @@
 #!/bin/bash
+# Terraform script for managing infrastructure on Azure
```
Comment (fmlisco) [inline src/domains/idpay-app/terraform.sh:2]: Last version of this script is v1.3
I suggest you to use a symbolic link. The original script(-s) is/are located into the following folder:
<git repo root>/scripts/

File: src/scripts/k8setup.sh [hunk 4]
```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env bash
```
Comment (fmlisco) [inline src/scripts/k8setup.sh:1]: Please use the scripts in the <git repo root>/scripts/ folder

**Review by fmlisco — CHANGES_REQUESTED** [review]

File: src/domains/idpay-common/04_cdn.tf (hunk 1 above)
Comment (mamari90) [reply src/domains/idpay-common/04_cdn.tf:257]: done

File: src/domains/idpay-common/04_cdn.tf (hunk 2 above)
Comment (mamari90) [reply src/domains/idpay-common/04_cdn.tf:252]: done

File: src/domains/idpay-app/terraform.sh (hunk 3 above)
Comment (mamari90) [reply src/domains/idpay-app/terraform.sh]: done

**Review by diegolagospagopa — APPROVED** [review]
👍

**Pull request merged**